installer/tue-install-impl.bash

```diff
@@ -1150,43 +1171,54 @@ function tue-install-ros
         return 0
     fi
 
-    if [ -z "$ROS_PACKAGE_INSTALL_DIR" ]
+    if [ "$install_type" != "git" ]
     then
-        tue-install-error "Environment variable ROS_PACKAGE_INSTALL_DIR not set."
```

Comment:
Why removing this check?

Reply:
This variable is defined in the script itself, so even if an environment variable named `ROS_PACKAGE_INSTALL_DIR` was set, this script would overwrite it. So this check is useless.